src/DefinitionResolver.php

```diff
@@ -134,7 +135,13 @@ public function createDefinitionFromNode(Node $node, string $fqn = null): Defini
         $def->parameters = [];
         if ($node instanceof Node\FunctionLike) {
             foreach ($node->getParams() as $param) {
-                $def->parameters[] = $this->prettyPrinter->prettyPrint([$param]);
+                if (!$param->getAttribute('parentNode')) {
```

Comment:
Every node except the root node should have a parentNode
Otherwise there must be a bug in NodeVisitor or ReferencesAdder